next-app/src/utils/hooks/useColors.ts

```diff
@@ -0,0 +1,102 @@
+import { useColorMode } from './useColorMode'
+
+const useColors = () => {
+  const { isDarkMode } = useColorMode()
+  const darkModeColors = {
+    'surface-dim': '#000000',
+    surface: '#1B181A',
+    'surface-bright': '#897981',
+    'surface-container-lowest': '#2A2528',
+    'surface-container-low': '#423C3F',
+    'surface-container': '#5F555A',
+    'surface-container-high': '#71646A',
+    'surface-container-highest': '#897981',
+    'font-primary': '#F1EFF0',
+    'font-secondary': '#A1929A',
+    'font-tertiary': '#897981',
+    'font-disabled': '#5F555A',
+    divider: '#5F555A',
+    varients: '#423C3F',
+    error: '#C26C7A',
+    'error-container': '#81271F',
+    'on-error': '#000000',
+    'on-error-container': ' #C26C7A',
+    'primary-50': '#451222',
+    'primary-100': '#732840',
+    'primary-200': '#892B49',
+    'primary-300': '#A63057',
+    'primary-400': '#C1416F',
+    'primary-500': '#D4608F',
+    'primary-600': '#E185AD',
+    'primary-700': '#ECAFCB',
+    'primary-800': '#F5D5E4',
+    'primary-900': '#F9EAF2',
+    'primary-1000': '#FBF4F7',
+    white: '#000000',
+    'white-fixed': '#ffffff',
+    'gray-50': '#2A2528',
+    'gray-100': '#423C3F',
+    'gray-200': '#5F555A',
+    'gray-300': '#71646A',
+    'gray-400': '#897981',
+    'gray-500': '#A1929A',
+    'gray-600': '#BAAFB4',
+    'gray-700': '#D4CDD1',
+    'gray-800': '#E9E5E7',
+    'gray-900': '#F1EFF0',
+    'gray-1000': '#F9F7F8',
+    black: '#FFFFFF',
+    'black-fixed': '#000000',
+  }
+
+  const lightModeColors = {
+    'surface-dim': 'rgba(0, 0, 0, 0.5)',
+    surface: '#ece9ea',
+    'surface-bright': '#ece9ea',
+    'surface-container-lowest': '#ffffff',
+    'surface-container-low': '#f9f7f8',
+    'surface-container': '#f1eff0',
+    'surface-container-high': '#e9e5e7',
+    'surface-container-highest': '#d4cdd1',
+    'font-primary': '#423c3f',
+    'font-secondary': '#71646a',
+    'font-tertiary': '#897981',
+    'font-disabled': '#a1929a',
+    divider: '#e9e5e7',
+    varients: '#f1eff0',
+    error: '#a12027',
+    'error-container': '#f9dbd7',
+    'on-error': '#ffffff',
+    'on-error-container': '#a12027',
+    'primary-50': '#fbf4f7',
+    'primary-100': '#f9eaf2',
+    'primary-200': '#f5d5e4',
+    'primary-300': '#ecafcb',
+    'primary-400': '#e185ad',
+    'primary-500': '#d4608f',
+    'primary-600': '#c1416f',
+    'primary-700': '#a63057',
+    'primary-800': '#892b49',
+    'primary-900': '#732840',
+    'primary-1000': '#451222',
+    white: '#ffffff',
+    'white-fixed': '#ffffff',
+    'gray-50': '#f9f7f8',
+    'gray-100': '#f1eff0',
+    'gray-200': '#e9e5e7',
+    'gray-300': '#d4cdd1',
+    'gray-400': '#baafb4',
+    'gray-500': '#a1929a',
+    'gray-600': '#897981',
+    'gray-700': '#71646a',
+    'gray-800': '#5f555a',
+    'gray-900': '#423c3f',
+    'gray-1000': '#2a2528',
+    black: '#000000',
+    'black-fixed': '#000000',
+  }
+
+  return isDarkMode ? darkModeColors : lightModeColors
```

Comment:
지금 그대로도 괜찮을거 같고 아래와 같은 형태로 내보내도 될거 같고요
근데 이거 아직 쓰는 곳 없는거죠?
```suggestion
  return {
    colorSet : isDarkMode ? darkModeColors : lightModeColors,
    isDarkMode,
  }
```

Reply:
넹 아직 쓰는 곳은 없고 쓰려고 만들었다가 필요 없어졌는데, 추후에 사용할 여지가 있어보여서 남겨놨어용